paasta_tools/cleanup_chronos_jobs.py

```diff
@@ -133,10 +134,18 @@ def filter_expired_tmp_jobs(client, job_names):
         )
         for job in temporary_jobs:
             last_run_time, last_run_state = chronos_tools.get_status_last_run(job)
+            chronos_job_config = chronos_tools.load_chronos_job_config(
+                service=service,
+                instance=instance,
+                cluster=cluster,
+                soa_dir=soa_dir,
+            )
+            interval = chronos_job_config.get_schedule_interval_in_seconds()
             if last_run_state != chronos_tools.LastRunState.NotRun:
                 if ((datetime.datetime.now(dateutil.tz.tzutc()) -
                      dateutil.parser.parse(last_run_time)) >
-                        datetime.timedelta(days=1)):
+                        datetime.timedelta(seconds=interval)):
```

Comment:
this means jobs with short interval are now removed much faster - do developers rely on having a day worth of history? maybe we want something like `max(1 day, job interval)` so we never go lower than 1 day?